JuliaInterface/julia/gaptypes.jl

```diff
@@ -82,6 +82,13 @@ struct GapFunc
     ptr::MPtr
 end
 
+"""
+    GapFEE
+
+> Holds a pointer to a GAP FFE immediate object
```

Comment:
```suggestion
> Holds a GAP FFE immediate object.
```
Could also change "Holds" to "Wraps" or so?